tests/test-input.xml

```diff
@@ -144,6 +145,7 @@
         </fuel_inrecipes>
         <fuel_outcommods>
           <val>waste</val>
+          <val>waste</val>
```

Comment:
This is odd to me. Why do we have two waste outcommodities? Is this an oddity in cycamore?

Reply:
we need an out commodity for each in commodity :)